projects/canopy/src/styles/mixins.scss

```diff
@@ -24,10 +24,11 @@
 ) {
   $box-shadow-inset-width: -0.063rem;
 
-  color: $default-color;
-  text-decoration: none;
   border-bottom: 0.125rem solid $default-color;
+  color: $default-color;
+  line-height: var(--link-line-height);
```

Comment:
Or should it just inherit from the parent?

Reply:
I tried that but it doesn't seem to work 🤔

Reply:
Here is using `inherit`: 
![Screenshot 2021-07-01 at 10 27 50](https://user-images.githubusercontent.com/8397116/124101231-0cff4d00-da57-11eb-8c76-2e432941753f.png)

Reply:
I was just wondering about a case where someone modifies the font-size of the parent with say, `lg-font-size-2` which contains a link, would that break the layout?

Reply:
BUT `initial` works so I'll change it to that 🤔

Reply:
Oh, I see. It is about where the "underline" sits... not sure initial is what we actually want. maybe the actual variable is the best solution

Reply:
Updated to `initial` now :)